### PR TITLE
Amls 5803 | Strip out TP's and RP questions from Pre application (update service)

### PR DESCRIPTION
**Pull request by slavvco**
As above removed 5 pages and all related code.

## Related / Dependant PRs?

AMLS-5879 - content work

## How Has This Been Tested?

- manually

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/controllers/declaration/DeclarationController.scala [hunk 1]
```diff
@@ -50,14 +50,12 @@ class DeclarationController @Inject () (val dataCacheConnector: DataCacheConnect
...
-              case ReadyForRenewal(_) if renewalComplete => Ok(
-                views.html.declaration.declare("declaration.declaration.amendment.title", "submit.renewal.application", name, false))
-              case ReadyForRenewal(_) | SubmissionReadyForReview | SubmissionDecisionApproved | RenewalSubmitted(_) => Ok(
-                views.html.declaration.declare("declaration.declaration.amendment.title", "submit.amendment.application", name, true))
+              case ReadyForRenewal(_) if renewalComplete =>
+                Ok(views.html.declaration.declare("declaration.declaration.amendment.title", "submit.renewal.application", name, false))
+              case ReadyForRenewal(_) | SubmissionReadyForReview | SubmissionDecisionApproved | RenewalSubmitted(_) =>
+                Ok(views.html.declaration.declare("declaration.declaration.amendment.title", "submit.amendment.application", name, true))
               case _ =>
-                println("statussssssssssss" + status)
```
Comment (ajchilts) [inline app/controllers/declaration/DeclarationController.scala:58]: Was there a 🐍 in there?

Comment (slavvco) [reply app/controllers/declaration/DeclarationController.scala:58]: upssssss

**Comment by slavvco** [comment]
@ajchilts I needed to do two changes after your review: fix the H1 in messages file (missed one letter) and remove clearing out the AddBusinessTypeFlowModel -> this line was causing problem when navigating back.

**Comment by slavvco** [comment]
code from this PR is now in AMLS-5879, therefore closing this PR

**Pull request closed without merging**